package.json

```diff
@@ -43,7 +43,9 @@
     "fbjs": "^0.8.4",
     "glamor": "^2.15.5",
     "inline-style-prefixer": "^2.0.4",
-    "lodash": "^4.15.0",
+    "is-function": "^1.0.1",
+    "is-plain-object": "^2.0.1",
+    "stylis": "^0.6.1",
```

Comment:
`stylis` belongs in https://github.com/styled-components/styled-components/pull/286 doesn't it?